releasenotes/notes/Allow-DBM-auto-discovery-users-to-optionally-set-region-d3006501542da7bf.yaml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
**❓ question:** ‏why there are 2 times the same release note ?

Reply:
ah jeeze, just a goof up. I deleted the old one because the name was too long, but obviously that didn't get pushed correctly. Fixing it now 🙇